tests/testthat/test-facets.R

```diff
@@ -47,3 +47,49 @@ test_that("3 facets becomes 3 panels", {
   u <- unique(trace.axes.df)
   expect_identical(nrow(u), 3L)
 })
+
+# expect a certain number of _unique_ [x/y] axes
```

Comment:
Great! This PR might be a good opportunity to rename `test-facets.R` into `test-ggplot-facets.R`.